integration/client/client_test.go

```diff
@@ -49,12 +50,31 @@ var (
 	noCriu       bool
 	supportsCriu bool
 	noShimCgroup bool
+	binaryName   = "runc"
+	runtimeOpts  NewContainerOpts
 )
 
 func init() {
 	flag.BoolVar(&noDaemon, "no-daemon", false, "Do not start a dedicated daemon for the tests")
 	flag.BoolVar(&noCriu, "no-criu", false, "Do not run the checkpoint tests")
 	flag.BoolVar(&noShimCgroup, "no-shim-cgroup", false, "Do not run the shim cgroup tests")
+
+	binaryNameEnv, valid := os.LookupEnv("RUNTIME_BINARY")
```

Comment:
Is there a reason this one needs to be different from the other configurations and use environment rather than flag?

Reply:
yes, we can use `flag.StringVar`.
But noticed there were env variables like `RUNC_FLAVOR`, so implemented using ENV.

Reply:
I think a CLI arg is better here.
We use `RUNC_FLAVOR` mostly in shell scripts (with a few exceptions to ignore tests).

Reply:
+1, and it'd be useful to make the argument something like `runc-flavor` too.  This could allow us to extend the integration tests later and target a different shim as well.

Reply:
Sure, I'll update the code to use args like `runc-flavor`.